src/main/resources/public/src/router.js

```diff
@@ -20,7 +20,7 @@ export default new Router({
           component: DataRequest
         },
         {
-            path: '/tovalidate',
+            path: '/request/validation/:id',
```

Comment:
the name is no so good because it implies you gonna validate. but you can reject too.

Reply:
`verify` is more appropriate ?